Source/CronosPlayUnreal/Public/PlayCppSdkActor.h

```diff
@@ -326,7 +359,124 @@ class CRONOSPLAYUNREAL_API APlayCppSdkActor : public AActor {
    * send wallet-connect information to unreal game thread
    *
    */
-  void sendEvent(const FWalletConnectSessionInfo &);
+  void sendEvent(FWalletConnectSessionInfo) const;
 
   static void destroyCoreClient();
+
+  /**
+   * Safely transfers `token_id` token from `from_address` to `to_address`.
+   * @param contractAddress erc721 contract
+   * @param fromAddress from address to move
+   * @param toAddress to address
+   * @param tokenid token id
+   * @param gasLimit gas limit
+   * @param gasPrice gas price
+   * @param Out FCronosSignedTransactionDelegate callback
+   */
+  UFUNCTION(BlueprintCallable,
+            meta = (DisplayName = "Erc721TransferFrom", Keywords = "Wallet"),
+            Category = "CronosPlayUnreal")
+  void Erc721TransferFrom(FString contractAddress, FString fromAddress,
+                          FString toAddress, FString tokenid, FString gasLimit,
+                          FString gasPrice,
+                          FCronosSignedTransactionDelegate Out);
+
+  /**
+   * erc721 Allows `approved_address` to withdraw from your account multiple
+   * times, up to the `amount` amount.
+   * @param contractAddress erc721 contract
+   * @param fromAddress from address
+   * @param approvedAddress  address to approve
+   * @param tokenid token id
+   * @param gasLimit gas limit
+   * @param gasPrice gas price
+   * @param Out FCronosSignedTransactionDelegate callback
+   *
+   */
+  UFUNCTION(BlueprintCallable,
+            meta = (DisplayName = "Erc721Approve", Keywords = "Wallet"),
+            Category = "CronosPlayUnreal")
+  void Erc721Approve(FString contractAddress, FString fromAddress,
+                     FString approvedAddress, FString tokenid, FString gasLimit,
+                     FString gasPrice, FCronosSignedTransactionDelegate Out);
+
+  /**
+   * erc1155 Moves `amount` tokens from `from_address` to `to_address` using the
+   * allowance mechanism.
+   * @param contractAddress erc1155 contract
+   * @param fromAddress from address to move
+   * @param toAddress to address
+   * @param tokenid token id
+   * @param amount amount
+   * @param additionaldata additional data
+   * @param gasLimit gas limit
+   * @param gasPrice gas price
+   * @param Out FCronosSignedTransactionDelegate callback
+   */
+  UFUNCTION(BlueprintCallable,
+            meta = (DisplayName = "Erc1155SafeTransferFrom",
+                    Keywords = "Wallet"),
+            Category = "CronosPlayUnreal")
+  void Erc1155SafeTransferFrom(FString contractAddress, FString fromAddress,
+                               FString toAddress, FString tokenid,
+                               FString amount, TArray<uint8> additionaldata,
+                               FString gasLimit, FString gasPrice,
+                               FCronosSignedTransactionDelegate Out);
+
+  /**
+   * erc1155 Allows `approved_address` to withdraw from your account multiple
```

Comment:
where is `approved_address`, should align the parameter name, here should be `approvedAddress`, elsewhere as well.

Reply:
ok